crm_helpdesk/report/crm_helpdesk_report_view.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!-- Helpdesk report after Sales Services Graph View -->
+        <record id="view_report_crm_helpdesk_graph" model="ir.ui.view">
+            <field name="name">crm.helpdesk.report.graph</field>
+            <field name="model">crm.helpdesk.report</field>
+            <field name="arch" type="xml">
+                <graph orientation="horizontal" string="Helpdesk" type="pivot" stacked="True">
```

Comment:
pivot view is not a graph
![image](https://user-images.githubusercontent.com/32061121/48712440-0dcc7300-ec0e-11e8-8910-4c9e7c7fe3c1.png)